core-compose/src/main/java/com/squareup/workflow/compose/ComposeWorkflow.kt

```diff
@@ -55,3 +57,22 @@ abstract class ComposeWorkflow<in PropsT, out OutputT : Any> :
   override fun asStatefulWorkflow(): StatefulWorkflow<PropsT, *, OutputT, ComposeRendering> =
     ComposeWorkflowImpl(this)
 }
+
+/**
+ * Returns a [ComposeWorkflow] that renders itself using the given [render] function.
+ */
+inline fun <PropsT, OutputT : Any> Workflow.Companion.composed(
```

Comment:
@rjrjr This is another friend for `Workflow.stateful()` and `Workflow.stateless()`.

Reply:
I forgot that `Workflow.stateful` even existed, only ever used the other one.